teps/0127-larger-results-via-sidecar-logs.md

```diff
@@ -0,0 +1,226 @@
+---
+status: implementable
+title: Larger Results via Sidecar Logs
+creation-date: '2022-11-30'
+last-updated: '2022-11-30'
+authors:
+- '@chitrangpatel'
+- '@jerop'
+see-also:
+- TEP-0086
+---
+
+# TEP-0127: Larger Results via Sidecar Logs
+
+<!-- toc -->
+- [Summary](#summary)
+- [Motivation](#motivation)
+  - [Goals](#goals)
+  - [Non-Goals](#non-goals)
+  - [Use Cases](#use-cases)
+- [Proposal](#proposal)
+  - [Feature Gate](#feature-gate)
+  - [Logs Access](#logs-access)
+  - [Size Limit](#size-limit)
+  - [PipelineRun Status](#pipelinerun-status)
+- [Design Details](#design-details)
+- [Design Evaluation](#design-evaluation)
+  - [Reusability](#reusability)
+  - [Simplicity](#simplicity)
+  - [Performance](#performance)
+  - [Security](#security)
+- [References](#references)
+<!-- /toc -->
+
+This TEP builds on the hard work of many people who have been tackling the problem over the past couple years,
+including but not limited to:
+- '@tlawrie'
+- '@imjasonh'
+- '@bobcatfish'
+- '@pritidesai'
+- '@tomcli'
+- '@ScrapCodes'
+- '@wlynch'
```

Comment:
Lifted these names from the authors of [TEP-0086](https://github.com/tektoncd/community/blob/main/teps/0086-changing-the-way-result-parameters-are-stored.md), please let me know if you'd like to be added or removed